frontend/integration-tests/protractor.conf.ts

```diff
@@ -158,10 +158,14 @@ export const config: Config = {
       'tests/operator-hub/operator-hub.scenario.ts',
       'tests/developer-catalog.scenario.ts',
       'tests/monitoring.scenario.ts',
+      'tests/devconsole/app-launcher.scenario.ts',
```

Comment:
Perspectives and app-launcher are core console functionality and not specific to dev console. I don't think this should be in the dev console sub-dir.

Reply:
I agree

Reply:
I'll update those values